HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1672,6 +1664,13 @@ def self.apply_detailed_performance_data_for_var_speed_systems(hpxml_bldg)
       if hvac_system.cooling_detailed_performance_data.empty?
         HVAC.set_cool_detailed_performance_data(hvac_system)
       else
+        # process capacity fraction of nominal
+        hvac_system.cooling_detailed_performance_data.each do |dp|
+          next unless dp.capacity.nil?
+
+          dp.capacity = dp.capacity_fraction_of_nominal * hvac_system.cooling_capacity
```

Comment:
Should also set `dp.capacity_isdefaulted = true` (and make sure `capacity_isdefaulted` is used in hpxml.rb).